rmgpy/data/kinetics/library.py

```diff
@@ -171,23 +171,23 @@ def convertDuplicatesToMulti(self):
         Merge all marked duplicate reactions in the kinetics library
         into single reactions with multiple kinetics.
         """
-        print "trying to find duplicates"
+        logging.info("trying to find duplicates")
         entries_to_remove = []
         for entry0 in self.entries.values():
             if entry0 in entries_to_remove:
                 continue
             reaction0 = entry0.item
             if not reaction0.duplicate:
                 continue
-            print "Found a duplicate reaction: {0}".format(reaction0)
+            logging.info("Found a duplicate reaction: {0}".format(reaction0))
             duplicates = [entry0]
             for entry in self.entries.values():
                 reaction = entry.item
                 if reaction0 is reaction:
                     continue
                 if reaction0.isIsomorphic(reaction, eitherDirection=False):
                     if reaction0.reversible != reaction.reversible:
-                        print "Reactions isomorphic but with different reversibilities"
+                        logging.info("Reactions isomorphic but with different reversibilities")
```

Comment:
I made the changes on my local branch, I'll push it to official after everything gets settled.